packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -117,6 +117,8 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
 
   const cameraPermissionGranted = props.cameraPermissionGranted;
   const micPermissionGranted = props.microphonePermissionGranted;
+  const isSafariBrowser = adapter.getState().environmentInfo?.environment.browser.toLowerCase() === 'safari';
```

Comment:
Could this instead be isSafariBrowser **&& speakers.length === 0**? My worry is one day safari will suddenly expose changing speakers independently